@here/harp-map-theme/resources/berlin_derived.json

```diff
@@ -15,11 +14,37 @@
         },
         "countryBorderLineWidth": {
             "type": "number",
-            "value": {
-                "interpolation": "Linear",
-                "zoomLevels": [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],
-                "values": [2000, 1400, 1000, 500, 220, 90, 50, 30, 20, 15, 10, 5, 2]
-            }
+            "value": [
```

Comment:
I wonder if we find an alternative way to `pretty print` our JSON theme files. The current `prettier` configuration that we use produces ugly results for interpolations.

Reply:
Not sure, prettier is  opinionaned  by design and intentionally rejects options to customiza layout more than they believe it's necessary. Brief look at https://prettier.io/docs/en/configuration.html didn't result in anything that may look like "that" option.